### ISSUE TITLE
Embedded Play docs should indicate which dependencies are needed

### ISSUE THREAD

**Issue by gmethvin** [playframework/playframework#6263]
If someone is using Play standalone in a maven or sbt project (see https://www.playframework.com/documentation/2.5.x/ScalaEmbeddingPlay), we should provide the dependency you need, e.g.

```scala
libraryDependencies += "com.typesafe.play" % "play_2.11" % "2.5.4"
```

And Java users will also need the `play-java` dependency

**Comment by schmitch** [comment]
Actually a server is needed aswell and I guess logback

**Comment by wsargent** [comment]
Any SLF4J implementation will do in theory.

**Comment by gmethvin** [comment]
@schmitch Yes, correct. Or the akka-http server. Updated the comment.

**Comment by gmethvin** [comment]
I think this is kind of a big deal because a lot of people are comparing play and akka-http and claiming Play is too bloated because it has all these features they don't need. Having a sample where they can include dependencies one by one would provide a more "fair" comparison for writing REST services.

**Comment by wsargent** [comment]
I did a dry run here:

https://github.com/wsargent/simplest-possible-play

**Comment by benmccann** [comment]
See also https://github.com/playframework/playframework/issues/5051

**Comment by gmethvin** [comment]
@benmccann @wsargent It seems you guys are taking the strategy of starting with the Play sbt plugin and disabling stuff. I'm suggesting just using Play as a library within another project. I think that's a better approach since you don't have to care about all the magic being done by Play's SBT plugins.

**Comment by benmccann** [comment]
@gmethvin no, i agree with you. my bug was basically saying we don't tell you how to use Play as a library within another project (though maybe sorry if that wasn't clear). you may still want some plugins though if not the main plugin. like perhaps you want to use the routes compiler
